src/calculateRentalCost.js

```diff
@@ -20,7 +20,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days >= 7) {
+    return sum - 50;
+  }
+
+  if (days >= 3) {
+    return sum - 20;
+  } else {
```

Comment:
no need for this `else`
```suggestion
  } 
```